readme.md

```diff
@@ -427,16 +432,18 @@ export default class BeerList {
 
-When beer table page is visited for the _first_ time, `onload()` method is going to be called. In our example, it performs an ajax request to fetch the data. This way, even if the page is then reload whilst on `/beers` or `/beers/23` the data is always going to be there to render.
+When beer table page is visited for the _first_ time, an `onload()` hook is called (if provided). In our example, it performs an ajax request to fetch the data. This way, even if the page is then reload whilst on `/beers` or `/beers/23` the data is always going to be there to render.
```

Comment:
why `hook` not `method`?

Reply:
In the sense that it is a special method. But I can see your point about possible connotation with react hooks - I'll change it to "special method"